README.md

```diff
@@ -91,89 +82,65 @@ The example above requires the following packages:
+  official OpenTelemetry SDK implementation. It contains Logging SDK, Metrics
+  SDK, and Tracing SDK.
+* [`opentelemetry-otlp`] exporter for sending logs, metrics and traces in the
+  OTLP format to an endpoint accepting OTLP, typically the OpenTelemetry
+  collector.
+* [`opentelemetry-stdout`] exporter for sending logs, metrics and traces to
+  stdout, for learning/debugging purposes.  
+* [`opentelemetry-http`] This crate contains utility functions to help with
+  exporting telemetry, propagation, over [`http`].
+* [`opentelemetry-appender-log`] This crate provides logging appender to route
+  logs emitted using the `log` crate to opentelemetry.
+* [`opentelemetry-appender-tracing`] This crate provides logging appender to route
+  logs emitted using the `tracing` crate to opentelemetry.  
+* [`opentelemetry-jaeger-propagator`] provides context propagation using [jaeger
+  propagation
+  format](https://www.jaegertracing.io/docs/1.18/client-libraries/#propagation-format).
 * [`opentelemetry-prometheus`] provides a pipeline and exporter for sending
-  metrics information to [`Prometheus`].
+  metrics to [`Prometheus`].
 * [`opentelemetry-semantic-conventions`] provides standard names and semantic
   otel conventions.
-* [`opentelemetry-stackdriver`] provides an exporter for Google's [Cloud Trace]
-  (which used to be called StackDriver).
-* [`opentelemetry-zipkin`] provides a pipeline and exporter for sending trace
-  information to [`Zipkin`].
-
-Additionally, there are also several third-party crates which are not
-maintained by the `opentelemetry` project. These include:
-
-* [`tracing-opentelemetry`] provides integration for applications instrumented
-  using the [`tracing`] API and ecosystem.
-* [`actix-web-opentelemetry`] provides integration for the [`actix-web`] web
-  server and ecosystem.
-* [`opentelemetry-application-insights`] provides an unofficial [Azure
-  Application Insights] exporter.
-* [`opentelemetry-tide`] provides integration for the [`Tide`] web server and
-  ecosystem.
-* [`trillium-opentelemetry`] provides metrics instrumentation for [`trillium`] http servers following [semantic-conventions](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md).
-
-If you're the maintainer of an `opentelemetry` ecosystem crate not listed
-above, please let us know! We'd love to add your project to the list!
-
-[`open-telemetry/opentelemetry-rust`]: https://github.com/open-telemetry/opentelemetry-rust
-[`opentelemetry-jaeger`]: https://crates.io/crates/opentelemetry-jaeger
-[`opentelemetry-otlp`]: https://crates.io/crates/opentelemetry-otlp
+* [`opentelemetry-zipkin`] provides a pipeline and exporter for sending traces
+  to [`Zipkin`].
+
+In addition, there are several other useful crates in the [OTel Rust Contrib
+repo](https://github.com/open-telemetry/opentelemetry-rust-contrib). A lot of
+crates maintained outside OpenTelemetry owned repos can be found in the
+[OpenTelemetry
```

Comment:
Do we have a followup to add the ecosystem crates to the registry?

Reply:
good point. Let me create an issue to add it.